src/appservice-kube/azext_appservice_kube/tests/latest/test_appservice_kube_scenario.py

```diff
@@ -18,6 +18,7 @@ class AppserviceKubernetesScenarioTest(ScenarioTest):
 
 # not lima-specific
 class WebappBasicE2EKubeTest(ScenarioTest):
+    @live_only()
```

Comment:
Why do you mark this test as `@live_only()`?

Reply:
This test passes locally (run live or as a recording), but fails in the CI with: `vcr.errors.CannotOverwriteExistingCassetteException`. I just pushed the recording for this test and removed the live_only and now the tests are failing

Reply:
> ERROR: Can't overwrite existing cassette ('/home/vsts/work/1/s/src/appservice-kube/azext_appservice_kube/tests/latest/recordings/test_linux_webapp_quick_create_kube.yaml') in your current record mode ('once').
No match for the request (<Request (POST) https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.Web/validate?api-version=2021-03-01>) was found.
Found 6 similar requests with 1 different matcher(s) :

> 1 - (<Request (POST) https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Web/checknameavailability?api-version=2021-03-01>).
Matchers succeeded : ['method', 'scheme', 'host', 'port', '_custom_request_query_matcher']
Matchers failed :
path - assertion failure :
/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.Web/validate != /subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Web/checknameavailability

This problem is caused by the fact that a rest request for validation is not recorded in the yaml file

Reply:
@StrawnSC I guess this may be because the code used in the local tests is not up to date. 
Could you pull the latest code from CLI main repo and CLI extension repo to the local, and then try to re-record the yaml file?

Reply:
@zhoxing-ms it's still failing even when I use the latest code for the main CLI (and rerun `azdev setup`)

Reply:
This testing issue seems unrelated to this change: the change was to `az functionapp create` whereas the test that passes locally but fails in the CI is testing `az webapp create`. Also, we've been getting requests from the function apps team to release this for a while. Can I go ahead and mark the test as `live_only` for now so we can release this?

Reply:
OK, if you are sure that this problem does not affect the feature of this PR, you can mark the test as `live_only` to avoid blocking the merging of PR. You can solve the test problems in the follow-up PR